File: interpret_python/interpret.py
def interpret(value, commands, args):
    for i in range(len(commands)):
        if commands[i] == "+":
            value = value + args[i]
        elif commands[i] == "-":
            value = value - args[i]
        elif commands[i] == "*":
            value = value * args[i]
        elif commands[i] == "/":
            value = value / args[i]
        elif commands[i] == "?":
            return -1

    return value

print(interpret(1,['+'],[1]))

File: interpret/interpret.java
import java.util.HashMap;
import java.util.Map;

class Test1{
  public static void main(String args[]){
    System.out.println(interpret(3,new String[]{"+","-"},new int[]{3,1}));
    System.out.println(interpret(1,new String[]{"+","*"},new int[]{1,3}));
  }
  public static int interpret(int value, String[] commands, int[] args) {

    for(int i = 0;i < commands.length;i++){
      switch(commands[i]) {
        case "+":
          value = value + args[i];
          break;
        case "-":
          value = value - args[i];
          break;
        case "*":
          value = value * args[i];
          break;
        case "/":
          value = value / args[i];
          break;
        case "?":
          return -1;
      }
    }
    return value;
  }
}
